### DISCUSSION TITLE
npm install tailwindcss gives error

### DISCUSSION THREAD

**Discussion by bishnu-47** [tailwindlabs/tailwindcss#3885] [Help] [answered]
i don't know whats happening, when trying
`npm install tailwindcss`

my npm version : **3.5.2**
my node version : **v8.10.0**

 i have attached the log file below

[npm-debug.log](https://github.com/tailwindlabs/tailwindcss/files/6236053/npm-debug.log)

**Comment by dance2die** [comment, marked as answer]
Tailwind CSS requires Node v.12.13.0 or higher.  

https://tailwindcss.com/docs/guides/laravel#setting-up-tailwind-css

**Comment by Smith8245** [comment]
What build Scripts are you talking about?

I'd advise you against using this project as your boilerplate. As far as I can see it's not providing you with anything besides some prebuild Views and outdated packages

**Comment by mjtbkh** [comment]
You are using an **out-dated version of node** and npm!
Go to [nodejs.org](https://nodejs.org/) and download the LTS release (v14.x) or simply run '**sudo apt update && apt upgrade**' in your terminal to update node to the latest stable version (**if you've installed it through apt**)
